lib/stripes-cli.js

```diff
@@ -21,19 +22,9 @@ updateNotifier({
   message: 'Update available - Refer to README.md:\nhttps://github.com/folio-org/stripes-cli',
 });
 
-// Monkey-patch Yargs to inform user of an unrecognized command when no suggestions could be provided
```

Comment:
I know this is incompatible with new versions of `yargs` but the downside to removing this is that if you type a command that `yargs` can't make a "did you mean?" suggestion, the CLI will just quietly exit and do nothing. My vote is that it's worth dropping this functionality to move forward (and perhaps file an enhancement request to `yargs`).